C_routing.md

```diff
@@ -276,8 +276,6 @@ The paths to the user facing reviews would look like a standard resource.
 /reviews
 /reviews/1234
 /reviews/1234/edit
-
-and so on
```

Comment:
This should probably be the fake ellipsis that we use throughout the rest of the guides.

`. . .`